test/StoragelessSessionTest/Http/SessionMiddlewareTest.php

```diff
@@ -45,14 +43,13 @@
      */
     public function testInjectsSessionDataEvenWithNoNextMiddleware(SessionMiddleware $middleware)
     {
-        $containerValue = uniqid('', true);
-
         $containerPopulationMiddleware = $this
-            ->buildFakeMiddleware(function (ServerRequestInterface $request) use ($containerValue) {
+            ->buildFakeMiddleware(function (ServerRequestInterface $request) {
                 /* @var $data Data */
                 $data = $request->getAttribute(SessionMiddleware::SESSION_ATTRIBUTE);
 
-                $data->set('foo', $containerValue);
+                $scope = $data->getScope('foo');
+                $scope->set('foo', \uniqid('', true));
```

Comment:
The point of using `$containerValue` from outside was to make sure the same value was checked in both middlewares